test/object.ts

```diff
@@ -160,6 +160,10 @@ test('object.instanceOf', t => {
 	t.throws(() => {
 		ow({unicorn: '🦄'}, ow.object.instanceOf(Unicorn));
 	}, 'Expected object `{"unicorn":"🦄"}` to be of type `Unicorn`');
+
+	t.throws(() => {
+		ow(null, ow.object.instanceOf(Unicorn));
+	}, 'Expected object `null` to be of type `Unicorn`');
```

Comment:
Needs a test for `undefined` too, just to be comprehensive.

Reply:
:+1: https://github.com/sindresorhus/ow/pull/211/commits/b1354c0d29421d27e5c01406950d5b77609e7445